cypress/e2e/CMS/$InfographicCreation.feature

```diff
@@ -0,0 +1,286 @@
+        When user selects test site section
+        And user fills out the following fields
+            | fieldLabel               | value                                                                        | field_name                     |
+            | Pretty URL               | home-and-landing-page-to-test-infographic                                    | field_pretty_url               |
+            | Page Title               | Automated Test Home and Landing page to test Infographic                     | title                          |
+            | Browser Title            | Automated Test Home and Landing page to test Infographic - Browser Title     | field_browser_title            |
+            | Card Title               | Automated Test Home and Landing page to test Infographic - Card Title        | field_card_title               |
+            | Meta Description         | Automated Test Home and Landing page to test Infographic - Meta Description  | field_page_description         |
+            | Feature Card Description | Automated Test Home and Landing page to test Infographic - Feature Card Desc | field_feature_card_description |
+        And user enters "Home and Landing page List Description" into "List Description" text field
+        And user clicks "Add Hero Banner" button for adding banner
+        And user uploads test file "heroimagenewsdesktop.jpg" as "Desktop" Version
+        And system waits for file upload process
+        And browser waits
+        And user uploads test file "heroimagenewstablet.jpg" as "Tablet" Version
+        And system waits for file upload process
+        And browser waits
+        And user uploads test file "heroimagenewsmobile.jpg" as "Mobile" Version
+        And system waits for file upload process
+        And browser waits
+        And user enters "This is to verify tagline" into "Tagline" text field in home and landing page
+        And user selects 3 Promotional Image from the list of images to be displayed in home and landing pages
+        And browser waits
+        And user remembers the source of selected promotional image to be displayed in home and landing pages for further verification
+        And browser waits
+        And user selects "Add Multimedia Row" from "Contents" dropdown in home and landing page
+        And browser waits
+        And user clicks on "Multimedia Card" link in the "Multimedia Row" text area in home and landing page
+        And browser waits
+        And user clicks on "Select media" button item
+        And browser waits
+        And user enters "Automated Test Infographic" into media search box and clicks "Select Infographic"
+        And user selects "Automated Test Infographic" item from media list
+        And browser waits
+        And user clicks on "Select media" button to select media item
+        And browser waits
+        And user remembers the name of media card item for future verification
```

Comment:
also, you know the name of infographic card - you created it yourself - so no needed here

Reply:
removed the step